code/script.js

```diff
@@ -246,98 +249,170 @@ const generateBoard = () => {
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  let randomElement = Math.random() * charactersInPlay.length //note to self! for a randomised number rounded off to length of array (will be a decimal (float) and therefore math.floor after)
+  let randomElementInt = Math.floor(randomElement); 
```

Comment:
Good form to change the initial logic, but knowing that the values of randomElement and randomElementInt are not being reassigned does two are more like const.